internal/operands/common-templates/reconcile_test.go

```diff
@@ -285,3 +361,24 @@ func getTestTemplates() []templatev1.Template {
 		},
 	}}
 }
+
+func getCommonTemplatesRestoredMetric() (string, float64) {
+	ch := make(chan prometheus.Metric, 1)
+	CommonTemplatesRestored.Collect(ch)
+	close(ch)
+	m := <-ch
+	dto := &io_prometheus_client.Metric{}
```

Comment:
What does `dto` mean? Can you use longer variable names?

Reply:
it is a commonly used name by Prometheus in these cases, but we can use `metric` for clarity